content/documentation/editingDomain/content.md

```diff
@@ -6,4 +6,178 @@ weight = 160
   sticky = true
 +++
 
-TODO Describe the model manager on a very high level, but mostly how it is integrated for a language with the model hub and which of the Model Hub capabilities are then covered generically
+#### General principles
+
+Edition of models is handled by the ModelManager, by executing Commands on one or several CommandStacks. The ModelManager is not directly exposed by the ModelHub API, so it is not (easily) possible to execute arbitrary commands on arbitrary models. Instead, the ModelManager is passed to ModelServiceContributions, which can then use it in their custom ModelService implementation to execute commands. This way, clients do not have to deal with Commands directly, but simply interact with the custom API.
+
+#### Model Service implementation
+
+As seen in the ModelHub section, the ModelServiceContribution is the entry point for language-specific contributions. Upon initialization of the ModelHub, each contribution will get access to the ModelManager, and should typically forward it to their ModelService implementation.
```

Comment:
`language-specific contributions` shouldn't we keep this more generic as `model-specific contributions` ?